mathics_scanner/tokeniser.py

```diff
@@ -9,6 +9,22 @@
 from mathics_scanner.errors import ScanError
 from mathics_scanner.prescanner import Prescanner
 
+ROOT_DIR = osp.dirname(__file__)
+try:
+    import ujson
+except ImportError:
+    import json as ujson  # type: ignore[no-redef]
+
+# Load Mathics3 character information from JSON. The JSON is built from
+# named-characters.yml
+
+operators_table_path = osp.join(ROOT_DIR, "data", "operators.json")
+assert osp.exists(
```

Comment:
This produces an error when the `operator.json` table is built for the first time. 
`mathics_scanner.generate.build_operator_tables` imports `mathics_scanner.__version__`,
with makes that `mathics_scanner.__init__` be loaded. But then, it tries to import this module, and finds that the file is not already created.

Reply:
A way to avoid this error would be to put all the initialization code inside an initialization function, and instead of raising an exception if the file does not exist, just show a warning.

Reply:
This was the situation before 370b8fe and this does not happen now Ubuntu and Macos. But I don't know why Windows is still failing here. 

I am exhausted at tracking down all the little inconsistencies for today. If can move this forward, please do.

Reply:
Sure!

Reply:
> A way to avoid this error would be to put all the initialization code inside an initialization function, and instead of raising an exception if the file does not exist, just show a warning.

Moving code to a function was done and delaying initialization was attempted but the code is too thorny that something in there is doing stuff earlier than when a tokenizer is first created.

I now have a workaround, so let's not add yet another.

Reply:
> Sure!

If you have work cycles to spare, here are some things that in my opinion are are more important than yet another workaround: 

* Put in the correct precedence for no-meaning operators
* Split out the following list of operator names by creating sections for left assoc infix, right assoc infix, flat infix,  (not yet done prefix/postfix), "misc" and newly added {Und,D]irectedEdge operators.

Reply:
> If you have work cycles to spare, here are some things that in my opinion are more important than yet another workaround:
> 
>     * Put in the correct precedence for no-meaning operators
Regarding this, "correct" would be in relation with the WMA `Precedence[...]`, isn´t it?  
> 
>     * Split out the following list of operator names by creating sections for left assoc infix, right assoc infix, flat infix,  (not yet done prefix/postfix), "misc" and newly added {Und,D]irectedEdge operators.

Do you mean as submodules of `no_meaning`?